db/db_impl/db_impl.h

```diff
@@ -437,6 +438,19 @@ class DBImpl : public DB {
   using DB::EndTrace;
   virtual Status EndTrace() override;
 
+  // Need a static version that can be called before DB::Open.
+  // It is used by rocksdb internal only.
+  static Status StartIOTrace(Env* env, const TraceOptions& options,
+                             std::unique_ptr<TraceWriter>&& trace_writer);
+
+  static Status EndIOTrace();
+
+  static std::shared_ptr<IOTracer> GetIOTracer() {
+    static IOTracer io_tracer;
```

Comment:
This means ```io_tracer``` is process scope and not DB. How would a user enable/disable per DB?